draft-ietf-rtcweb-jsep.xml

```diff
@@ -3077,7 +3081,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             on a per-codec basis (e.g., if the offerer somehow offered
             support for CN but set "usedtx=0" for Opus, setting
             VoiceActivityDetection to true would result in an answer
-            with CN codecs and "usedtx=0").</t>
+            with CN codecs and "usedtx=0"). The impact of this rule is that
+            an answerer will not offer silence suppression with any
```

Comment:
The use of the word 'offer' here in an answer context is somewhat confusing.

Perhaps 'offer' -> 'try to use'